docs/log-replicator/routing_queues.md

```diff
@@ -0,0 +1,121 @@
+## Log Replication via Routing Queues & Destination stream tags
+
+[Primary Requirements](#primary-requirements)
+
+[Routing Queues & Stream Tags](#routing-queues-and-destination-stream-tags)
+
+[LR Client on Sender](#client-on-sender)
+
+[LR Server on Sender](#log-entry-reader-in-server-on-sender)
+
+[LR Full Snapshot Sync Sender](#full-sync-sender)
+
+[LR Server on Receiver](#lr-server-on-sink)
+
+[LR Client on Receiver](#lr-client-on-sink)
+
+### Primary requirements
+When a transaction touches a record on the source cluster that mutation
+needs to be disseminated to one or more remote sites.
+The data also needs to be modified after dissemination on the sink cluster.
+
+Client will provide one or more opaque payloads with a list of destinations in a transaction.
+LR client on the sender intercepts this in the same transaction, transforms & persists this in corfu.
+
+**If a payload needs to go to multiple sites, that payload must not be duplicated since there can be
+several sites.**
+The payload is opaque to LR, the receiver is the one that needs to read this data and apply the updates to various tables.
+
+### Routing Queues and Destination Stream tags
+LR Server on source and sink will have one session per remote destination.
+Data from the clients will be written into the following well known Routing Queues
+1. `LRQ_Send_LogEntries`: This is the one shared queue on the sender where delta updates are placed transactionally.
+2. `LRQ_Send_SnapSync`: This is the one shared queue on the sender where full snapshot sync updates are placed.
+3. `LRQ_Recv_<remote_id>`: This is the queue where the LR on Sink places the updates arriving from the remote source.
+
+Each update made carries destinations and is also tagged with a destination specific stream tag with the following convention:
+1. `lrq_logentry_<remote_id>`: Destination specific stream tag applied when entries are placed into `LRQ_Send_LogEntries`
+2. `lrq_snapsync_<remote_id>`: Destination specific stream tag applied when entries are placed into `LRQ_Send_SnapSync`
+3. `lrq_recv`: This is the tag applied on the sink by LR server to all updates be it full sync or snapshot sync.
```

Comment:
"full sync or snapshot sync" -> "log entry sync or snapshot sync"?

Reply:
thank you - yes i meant log entry sync the second time